### PR TITLE
Added asset connection setup

### PR DESCRIPTION
**Pull request by lcpopa**
Signed-off-by: Liviu Popa <liviu-constantin.popa@ing.com>

### CODE REVIEW

File: egeria-connector-jdbc/src/main/java/org/odpi/openmetadata/adapters/connectors/integration/jdbc/transfer/DatabaseConnectionConsumer.java [hunk 1]
```diff
@@ -0,0 +1,173 @@
+package org.odpi.openmetadata.adapters.connectors.integration.jdbc.transfer;
```
Comment (planetf1) [inline egeria-connector-jdbc/src/main/java/org/odpi/openmetadata/adapters/connectors/integration/jdbc/transfer/DatabaseConnectionConsumer.java:1]: Add SPDX headers

File: egeria-connector-jdbc/src/main/java/org/odpi/openmetadata/adapters/connectors/integration/jdbc/ffdc/JdbcConnectorAuditCode.java [hunk 2]
```diff
@@ -42,12 +42,12 @@ public enum JdbcConnectorAuditCode implements AuditLogMessageSet {
             "Consult logs for further details"),
     ERROR_READING_JDBC("JDBC-CONNECTOR-0005",
             OMRSAuditLogRecordSeverity.EXCEPTION,
-            "An sql exception was received by method {0}. Exception message is {1}",
+            "An sql exception was received by method {0}. Exception message is: {1}",
```
Comment (planetf1) [inline egeria-connector-jdbc/src/main/java/org/odpi/openmetadata/adapters/connectors/integration/jdbc/ffdc/JdbcConnectorAuditCode.java:45]: usually sql is capitalised ie SQL

File: egeria-connector-jdbc/src/main/java/org/odpi/openmetadata/adapters/connectors/integration/jdbc/transfer/DatabaseConnectionConsumer.java [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+    private final AuditLog auditLog;
+    private final JdbcMetadata jdbcMetadata;
+
+    DatabaseConnectionConsumer(DatabaseIntegratorContext databaseIntegratorContext, AuditLog auditLog, JdbcMetadata jdbcMetadata){
+        this.databaseIntegratorContext = databaseIntegratorContext;
+        this.auditLog = auditLog;
+        this.jdbcMetadata = jdbcMetadata;
+    }
+
+    @Override
```
Comment (planetf1) [inline egeria-connector-jdbc/src/main/java/org/odpi/openmetadata/adapters/connectors/integration/jdbc/transfer/DatabaseConnectionConsumer.java:38]: public methods should supply javadoc

File: egeria-connector-jdbc/src/main/java/org/odpi/openmetadata/adapters/connectors/integration/jdbc/transfer/JdbcMetadataTransfer.java [hunk 4]
```diff
@@ -61,6 +60,8 @@ public boolean execute() {
                         EXITING_ON_METADATA_TRANSFER.getMessageDefinition());
                 return false;
             }
+            createAssetConnection(databaseElement);
```
Comment (planetf1) [inline egeria-connector-jdbc/src/main/java/org/odpi/openmetadata/adapters/connectors/integration/jdbc/transfer/JdbcMetadataTransfer.java:63]: public methods should have javadoc

File: egeria-connector-jdbc/src/main/java/org/odpi/openmetadata/adapters/connectors/integration/jdbc/transfer/DatabaseConnectionConsumer.java (hunk 3 above)
Comment (planetf1) [reply egeria-connector-jdbc/src/main/java/org/odpi/openmetadata/adapters/connectors/integration/jdbc/transfer/DatabaseConnectionConsumer.java:38]: (applies to other files also)

**Review by planetf1 — CHANGES_REQUESTED** [review]

File: egeria-connector-jdbc/src/main/java/org/odpi/openmetadata/adapters/connectors/integration/jdbc/transfer/DatabaseConnectionConsumer.java (hunk 3 above)
Comment (lcpopa) [reply egeria-connector-jdbc/src/main/java/org/odpi/openmetadata/adapters/connectors/integration/jdbc/transfer/DatabaseConnectionConsumer.java:40]: Javadoc will soon follow

File: egeria-connector-jdbc/src/main/java/org/odpi/openmetadata/adapters/connectors/integration/jdbc/transfer/JdbcMetadataTransfer.java (hunk 4 above)
Comment (lcpopa) [reply egeria-connector-jdbc/src/main/java/org/odpi/openmetadata/adapters/connectors/integration/jdbc/transfer/JdbcMetadataTransfer.java:63]: Javadoc will soon follow

**Pull request merged**